android/res/values-pl/strings.xml

```diff
@@ -209,6 +209,10 @@
 	<string name="pref_map_3d_buildings_title">Budynki w 3D</string>
 	<!-- Settings «Route» category: «Tts enabled» title -->
 	<string name="pref_tts_enable_title">Komunikaty głosowe</string>
+	<!-- Settings «Route» category: «Tts announce street names» title -->
+	<string name="pref_tts_street_names_title">Oznajmianie nazw ulic</string>
```

Comment:
@lunarna-gh what about "Wymawiaj nazwy ulic" or if you have better translation?

Reply:
`Wymawianie nazw ulic` fits other strings in the settings better

Reply:
Consistency is a good thing but until we find a better translation let's keep it like I said.

Reply:
?? It is the same translation